update-js.sh

```diff
@@ -10,4 +10,43 @@ mkdir -p $dest
 cp node_modules/swagger-ui-dist/swagger-ui-bundle.js $dest
 cp node_modules/swagger-ui-dist/swagger-ui-standalone-preset.js $dest
 cp node_modules/swagger-ui-dist/swagger-ui.css $dest
+
+dest=src/Bridge/Symfony/Bundle/Resources/public/es6-promise/
+
+yarn add --production --no-lockfile es6-promise
+if [ -d $dest ]; then
+  rm -Rf $dest
+fi
+mkdir -p $dest
+cp node_modules/es6-promise/dist/es6-promise.auto.min.js $dest
+
+dest=src/Bridge/Symfony/Bundle/Resources/public/fetch/
+
+yarn add --production --no-lockfile fetch
+if [ -d $dest ]; then
+  rm -Rf $dest
+fi
+mkdir -p $dest
+cp node_modules/fetch/lib/fetch.js $dest
+
+dest=src/Bridge/Symfony/Bundle/Resources/public/react/
+
+yarn add --production --no-lockfile react react-dom
+if [ -d $dest ]; then
+  rm -Rf $dest
+fi
+mkdir -p $dest
+cp node_modules/react/dist/react.min.js $dest
+cp node_modules/react-dom/dist/react-dom.min.js $dest
+
+dest=src/Bridge/Symfony/Bundle/Resources/public/graphiql/
+
+yarn add --production --no-lockfile graphiql
+if [ -d $dest ]; then
+  rm -Rf $dest
+fi
+mkdir -p $dest
+cp node_modules/graphiql/graphiql.min.js $dest
+cp node_modules/graphiql/graphiql.css $dest
+
```

Comment:
gosh :cat: maybe we should kinda create a js repository for all the graphql stuff in the future and just une one module for all these.

This also means that we're now publishing react sources in the source code of api-platform, totally :-1: on this personally...

Reply:
Creating our own JS repo will not help. AFAIK, it's not possible to upload artifacts on Packagist, so the JS must be included in the repo. It's still a dependency, the way it is downloaded doesn't change anything from a legal POV.

This script can be dramatically improved but we can still handle that later